src/Uno.UI/UI/Xaml/Controls/MediaPlayerElement/MediaTransportControls.cs

```diff
@@ -1220,53 +1097,50 @@ private void UpdateVolumeMuteStates(bool isExplicitMuteToggle = false, bool useT
 			var state = isMuted
 				? VisualState.VolumeMuteStates.MuteState
 				: VisualState.VolumeMuteStates.VolumeState;
-			VisualStateManager.GoToState(this, state, useTransition);
+			VisualStateManager.GoToState(this, state, useTransitions);
 
 			var uiaKey = isMuted
 				? UIAKeys.UIA_MEDIA_UNMUTE
 				: UIAKeys.UIA_MEDIA_MUTE;
 			SetAutomationNameAndTooltip(m_tpMuteButton, uiaKey);
 		}
-
-		private void UpdateFullWindowStates(bool useTransition = true)
+		private void UpdateFullWindowStates(bool useTransitions = true)
 		{
 			if (_mpe is not null)
 			{
 				var state = _mpe.IsFullWindow
 					? VisualState.FullWindowStates.FullWindowState
 					: VisualState.FullWindowStates.NonFullWindowState;
-				VisualStateManager.GoToState(this, state, useTransition);
+				VisualStateManager.GoToState(this, state, useTransitions);
 
 				var uiaKey = _mpe.IsFullWindow
 					? UIAKeys.UIA_MEDIA_EXIT_FULLSCREEN
 					: UIAKeys.UIA_MEDIA_FULLSCREEN;
 				SetAutomationNameAndTooltip(m_tpFullWindowButton, uiaKey);
 			}
 		}
-
-		private bool IsMediaPlayerLoopingEnabled =>
-			ApiInformation.IsPropertyPresent(typeof(Windows.Media.Playback.MediaPlayer).FullName!, nameof(Windows.Media.Playback.MediaPlayer.IsLoopingEnabled))
-					&& (_mpe?.MediaPlayer.IsLoopingEnabled ?? false);
```

Comment:
We need to implement the property on all platforms to avoid exceptions.

Reply:
i kept the ApiInfo guards without a helper property
can we keep it as it is for this pr? 
or, just impl them as a noop `{ get; set; }` property for now on the missing platforms with issues to follow up?